### PR TITLE
Add files via upload

### PR DESCRIPTION
**Pull request by link1697**
Added Geoschem model;
I have reviewed and hereby sign the CLA

### CODE REVIEW

File: src/Lowfunctions.jl [hunk 1]
```diff
@@ -0,0 +1,452 @@
+"""
```
Comment (ctessum) [inline src/Lowfunctions.jl:1]: What does the "low" mean in "LowFunctions.jl"? Should it be "Law" instead?

**Comment by ctessum** [comment]
recheck

**Review by ctessum — COMMENTED** [review]
Looking good! I mainly just have a question about the name of one of the files.

File: src/Lowfunctions.jl (hunk 1 above)
Comment (link1697) [reply src/Lowfunctions.jl:1]: Yes, it should be Law.

**Review by link1697 — COMMENTED** [review]
I have reviewed and hereby sign the CLA

**Review by link1697 — COMMENTED** [review]
I have reviewed and hereby sign the CLA

**Review by link1697 — COMMENTED** [review]
I have reviewed and hereby sign the CLA

**Comment by link1697** [comment]
I have reviewed and hereby sign the CLA

**Review by ctessum — APPROVED** [review]
Looks good!

**Pull request merged**